browser/scripts/ui-core.js

```diff
@@ -107,38 +109,47 @@ VizorUI.prototype.setupStateStoreEventListeners = function() {
 				.toggleClass('ui_on', inProgramMode)
 				.toggleClass('ui_off', inBuildMode);
 
-			if (inProgramMode) {
-				dom.tabObjects.addClass('inactive ui_off');
-				dom.tabPresets.removeClass('inactive ui_off');
-			} else {
-				dom.tabObjects.removeClass('inactive ui_off');
-				dom.tabPresets.addClass('inactive ui_off');
-			}
+			// LIs
+			dom.tabObjects
+				.toggleClass('disabled', inProgramMode)
+				.toggleClass('ui_off', inProgramMode)
+				.toggleClass('ui_on', inBuildMode)
+
+			dom.tabPresets
+				.toggleClass('disabled', inBuildMode)
+				.toggleClass('ui_off', inBuildMode)
+				.toggleClass('ui_on', inProgramMode)
+
+			dom.btnMove.attr('disabled',!inBuildMode);
+			dom.btnScale.attr('disabled',!inBuildMode);
+			dom.btnRotate.attr('disabled',!inBuildMode);
+
+			setTimeout(function(){
+				if (inBuildMode) dom.tabObjects.find('a').trigger('click')
+				else if (inProgramMode) dom.tabPresets.find('a').trigger('click')
+			}, 100);
```

Comment:
why setTimeout? is the idea to do this in the next tick? (setTimeout with 0ms does that)